index.js

```diff
@@ -0,0 +1,17 @@
+export { default as Container } from './src/container/Container';
```

Comment:
Maybe move this into src, then point the main from package.json at dist/index.js ?

We also need to export the configure function from globals.js